src/server/game/Warden/WardenWin.cpp

```diff
@@ -239,6 +252,80 @@ void WardenWin::HandleHashResult(ByteBuffer& buff)
     _initialized = true;
 }
 
+uint16 WardenWin::GetFreePayloadId()
+{
+    uint16 payloadId = _wardenPayloadOffset;
+
+    while (CachedChecks.find(payloadId) != CachedChecks.end())
+    {
+        payloadId++;
+    }
+
+    return payloadId;
+}
+
+/**
+* Register a payload into cache and returns its payload id.
+* @param payload The payload to be stored in WardenWin::CachedChecks.
+* @return uint16 The payload id for use with Warden::QueuePayload.
+*/
```

Comment:
```suggestion
/**
* @brief Register a payload into cache and returns its payload id.
* @param payload The payload to be stored in WardenWin::CachedChecks.
* @return uint16 The payload id for use with Warden::QueuePayload.
*/
```
add @brief to the first line for all Doxy comments please. Also I love that you use doxy comments <3

Reply:
Fixed, thanks :)